streams/si.m3u

```diff
@@ -11,8 +11,6 @@ http://91.220.221.60/gtv_hls/gtv_03.m3u8
 https://data.siptv.si/live-siptv/hls/index.m3u8
 #EXTINF:-1 tvg-id="TrzicTV.si",Tržič TV (1080p) [Not 24/7]
 https://vpn.video2go.live:444/trzic/tv/playlist.m3u8
-#EXTINF:-1 tvg-id="TVDK.sk",TVDK (486p) [Not 24/7]
-https://5ca49f2417d90.streamlock.net/live/dolnykubin1/playlist.m3u8
```

Comment:
Working
<img width="1406" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/47c34066-7c98-413f-aede-114db1a47155">

Reply:
Just moved it to sk.m3u